server/validation/validateEmail.js

```diff
@@ -0,0 +1,8 @@
+const yup = require('yup');
+
+module.exports = yup.object({
+  email: yup
+    .string()
+    .email()
+    .required(),
+});
```

Comment:
you could've simply used `yup.string.ema....` directly without using `yup.object`